_projects/2024/100535255/100535255.Rmd

```diff
@@ -0,0 +1,1059 @@
+---
+title: "Degrees and Nobels: A non-essential link"
+description: |
+  A comprehensive look at Nobel Prizes from 1901 to 2012, uncovering patterns 
+  in laureates ages, trends across categories, educational qualifications, academic 
+  affiliations and hometown demographics.
+categories: "2024"
+author: "Andrea Rivera-Mateos"
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="200%", fig.align="center")
+```
+
+```{css text alignment}
+#| echo: false
+p {text-align: justify}
+```
+
+## **Setting the stage**
```

Comment:
Also, please remove any additional formatting from titles (things like bold notation, italics...). To preserve a uniform style, titles should look as the CSS of the page tells them.